odk1-src/collect-settings.rst

```diff
@@ -183,6 +183,18 @@ To access Form Management settings:
 
 .. rubric:: Form update
 
+:guilabel:`Blank form update mode`
+"""""""""""""""""""""""""""""""""""
+
+  Specifies how blank forms should be updated:
+
+  :guilabel:`Manually`
+    Enumerators will have to manually manage blank forms on the device using :guilabel:`Get Blank Form` and :guilabel:`Delete Saved Form`.
+  :guilabel:`Previously downloaded forms only`
+    The enumerator will receive a notification when one or more forms already downloaded have an update available to their form definition or media files. Tapping on the notification will go to :guilabel:`Get Blank Form` where the user can choose to download some or all of the updated forms.
+  :guilabel:`Exactly match server`
+    Collect will automatically download and update forms based on what's on the server. In addition, forms not on the server will be deleted from Collect. This mode hides :guilabel:`Get Blank Form` and the :guilabel:`Blank Forms` tab in :guilabel:`Delete Saved Form` as they are not required. The enumerator is still able to manually update forms with the server on the :guilabel:`Fill Blank Form` screen.
```

Comment:
`The enumerator is still able to manually update` makes it sound like they can pick forms. Maybe `The enumerator can trigger an update from the server on...`